dolphinscheduler-ui/src/views/datasource/list/use-form.ts

```diff
@@ -137,24 +159,52 @@ export function useForm(id?: number) {
     } else {
       state.showPrincipal = false
     }
+    if (state.detailForm.id === undefined) {
+      await getSameTypeTestDataSource()
```

Comment:
It seems to be no need to reload the test options here. The test options need to be reloaded only when the test flag is changed to 0.

Reply:
need

Reply:
To load when changing type.